### DISCUSSION TITLE
Win10Pro SumatraPDF 3.4.6 64-bit no longer opens PDF file when Sumatra opened again

### DISCUSSION THREAD

**Discussion by Bub2480** [sumatrapdfreader/sumatrapdf#3342] [Q&A]
A week ago I updated to 22H2 of Win10Pro x64.  Today is the first time opening SumatraPDF since.

Before, if I opened 1.pdf and closed SumatraPDF WITHOUT closing the file, next time I opened SumatraPDF 1.pdf would be opened again.  That no longer works.  

I've restarted twice today.  When I open advanced settings file, I have 
RememberOpenedFiles = true
RememberStatePerDocument = true
RestoreSession = true

In options, UNCHECKED Remember opened files.  Closed Sumatra.  Opened Sumatra.  CHECKED Remember opened files.  Closed Sumatra.  Opened Sumatra.  Opened 1.pdf.  Closed Sumatra.  Opened Sumatra.  NO PDF FILES OPENED WITH PROGRAM.  1.pdf is in a local folder on a local hard drive, so no web connectivity in play.  

I've tested different pdfs in different folders, same issue.  

If I look in Settings, File, 1.pdf and 2.pdf are in the list.  So it remembers that I had those opened.  

Interestingly, if I open 1.pdf AND 2.pdf, then close Sumatra, open Sumatra, it will have BOTH PDFs opened with Sumatra, but will NOT do that for either file just one at a time.  

I searched the discussions for anything related back to 2022-10, did not find anything relevant.

**Comment by GitHubRulesOK** [comment]
look at help about in SumatraPDF and check version
older versions would only remember a session e.g. 2 or more files at time of closure and NOT one.
However that was changed more recently (just cant remember which recent version, release or not?)

**Comment by Bub2480** [comment]
I have been using SumatraPDF since 3.2 dated 2020-03-15.  After initially installing and adjusting settings, this has ALWAYS remembered PDFs that were opened when SumatraPDF was closed (and NOT closed the individual files), regardless of whether 1 or 3 or 5 or …..  

 

I downloaded 3.4.6 on 8-14-22, and assume installed that day.  The behavior did not change until the last couple of weeks.  While I don’t open PDFs and want them to remain open frequently, it is a feature I have always used and relied on.  If, for example, some Windows or other update requires a reboot, and I was working with 1 or more PDFs, this saves me from having to write down what I had open.  After the reboot, I can just open SumatraPDF and all my files are still open.  

 

I have always used the 64-bit version, and always installed it, as opposed to a portable use.

**Reply by GitHubRulesOK** [reply]
historically since after 2014 https://github.com/sumatrapdfreader/sumatrapdf/issues/28#issuecomment-66934824 SumatraPDF would NOT remember one tab still had that policy in 2017 https://github.com/sumatrapdfreader/sumatrapdf/issues/458#issuecomment-301298010 and in 2018-2022 https://github.com/sumatrapdfreader/sumatrapdf/issues/1055#issuecomment-411536490
however that changed around about https://github.com/sumatrapdfreader/sumatrapdf/discussions/2862#discussioncomment-3399145 i.e https://github.com/sumatrapdfreader/sumatrapdf/issues/1055#issuecomment-1149436659

Thus that was the designed behaviour up to 3.4.6 (2022-06-08) and AFAIK only changed after, from above:-
>this is now fixed in pre-release builds
